packages/core/src/api/blockManipulation/tables/tables.ts

```diff
@@ -185,7 +183,7 @@ type OccupancyGrid = (RelativeCellIndices & {
  * @returns an {@link OccupancyGrid}
  */
 export function getTableCellOccupancyGrid(
-  block: BlockFromConfigNoChildren<DefaultBlockSchema["table"], any, any>,
```

Comment:
Stopped exporting `BlockFromConfigNoChildren` so we have more consistency in the codebase

Reply:
While we are cleaning up types, I believe that:

```ts
BlockFromConfig<DefaultBlockSchema["table"], any, any>
```

Should only be:

```ts
BlockFromConfig<DefaultBlockSchema["table"]>
```

Most of the time we don't care about the other parameters, and have to remove their defaults with any. This goes for most instances of B, I, S